CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 
 set(MIR_VERSION_MAJOR 2)
 set(MIR_VERSION_MINOR 6)
-set(MIR_VERSION_PATCH 0)
+set(MIR_VERSION_PATCH 1)
```

Comment:
I think we've decided this should be 2.7.0, not 2.6.1, right? It's got feature additions.

Regardless, this should match what's in debian/changelog